lib/fastlane/plugin/revenuecat_internal/actions/update_hybrids_versions_file_action.rb

```diff
@@ -19,9 +19,25 @@ def self.run(params)
         ios_version = Helper::UpdateHybridsVersionsFileHelper.get_ios_version_for_hybrid_common_version(hybrid_common_version)
         UI.message("Obtained ios version #{ios_version} for PHC version #{hybrid_common_version}")
 
+        billing_client_version = Helper::UpdateHybridsVersionsFileHelper.get_android_billing_client_version(android_version)
+        UI.message("Obtained android billing client version #{billing_client_version} for PHC version #{hybrid_common_version}")
+
         File.open(versions_file_path, 'r+') do |file|
           lines = file.each_line.to_a
-          lines.insert(2, "| #{new_sdk_version} | #{ios_version} | #{android_version} | #{hybrid_common_version} |\n")
+          if lines[0].split('|').length <= 6
+            lines[0] = "#{lines[0].strip} Android Billing Client version |\n"
+            lines[1] = "#{lines[1].strip}--------------------------------|\n"
+            lines.each_with_index do |line, index|
+              if index > 1
+                lines[index] = "#{line.strip} |\n"
+              end
+            end
+          end
+          lines.insert(2, "| #{new_sdk_version} " \
+                          "| [#{ios_version}](https://github.com/RevenueCat/purchases-ios/releases/tag/#{ios_version}) " \
+                          "| [#{android_version}](https://github.com/RevenueCat/purchases-android/releases/tag/#{android_version}) " \
+                          "| [#{hybrid_common_version}](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/#{hybrid_common_version}) " \
+                          "| [#{billing_client_version}](https://developer.android.com/google/play/billing/release-notes) |\n")
```

Comment:
I think you can avoid explicitly adding the ` | ` separator by using `Array::join`, like: 

```ruby
new_line = [
    new_sdk_version,
    "[#{ios_version}](https://github.com/RevenueCat/purchases-ios/releases/tag/#{ios_version})",
    "[#{android_version}](https://github.com/RevenueCat/purchases-android/releases/tag/#{android_version})",
    "[#{hybrid_common_version}](https://github.com/RevenueCat/purchases-hybrid-common/releases/tag/#{hybrid_common_version})",
    "[#{billing_client_version}](https://developer.android.com/google/play/billing/release-notes)"
  ].join(' | ')
```

Reply:
can probably use it for the previous section too